media/base/mime_util_internal.cc

```diff
@@ -432,18 +432,16 @@ void MimeUtil::AddContainerWithCodecs(std::string mime_type, CodecSet codecs) {
 }
 
 bool MimeUtil::IsSupportedMediaMimeType(base::StringPiece mime_type) const {
-#if BUILDFLAG(IS_COBALT)
 #if BUILDFLAG(USE_STARBOARD_MEDIA)
   SbMediaSupportType support_type =
       SbMediaCanPlayMimeAndKeySystem(mime_type.data(), "");
   bool result = support_type != kSbMediaSupportTypeNotSupported;
   LOG(INFO) << __func__ << "(" << mime_type << ") -> "
             << (result ? "true" : "false");
   return result;
-#endif  // BUILDFLAG(USE_STARBOARD_MEDIA)
-#else   // BUILDFLAG(IS_COBALT)
+#else  // BUILDFLAG(USE_STARBOARD_MEDIA)
   return media_format_map_.contains(base::ToLowerASCII(mime_type));
-#endif  // BUILDFLAG(IS_COBALT)
```

Comment:
Just a note that I recall Chromium style wants two spaces before  
an in-line comment, so the LHS is OK. Did `git cl format` do this?  
Or maybe it was the presubmit...?

Reply:
Ran these through git cl format, unfortunately that doesn't happen automatically yet